sender.py

```diff
@@ -0,0 +1,24 @@
+from generate_temperature import generate_temperature_samples
+from generate_charge_rate import generate_charge_rate_samples
+
+
+def main():
+    # Totally 10 samples, each sample consisting of 60 readings for each parameters
+    NUM_OF_READINGS = 60
+    NUM_OF_SAMPLES = 10
+
+    # Outputting the Charge rates reading to standard output console
+    for _ in range(NUM_OF_SAMPLES):
```

Comment:
As per the requirement, it expects to send only 50 readings in a stream, why 10*60 readings are sent for each parameter? can be corrected as per the requirement

Reply:
As per requirement , it is said to send "fifty readings in a stream" . I am sending multiple streams using the "NUM_OF_SAMPLES" parameter and each stream I am sending 60 readings. I guess this can be configurable.